debugger/debugrt/host/host.c

```diff
@@ -264,6 +282,79 @@ oe_result_t oe_debug_notify_enclave_terminated(oe_debug_enclave_t* enclave)
     return result;
 }
 
+oe_result_t oe_debug_notify_module_loaded(oe_debug_module_t* module)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    bool locked = false;
+
+    if (module == NULL || module->magic != OE_DEBUG_MODULE_MAGIC ||
+        module->enclave == NULL)
+    {
+        result = OE_INVALID_PARAMETER;
```

Comment:
The reason we cannot use OE_RAISE is that `oedebugrt` is intended to be compiled into a separate shared library on Windows. `OE_RAISE` internally uses `oe_log` which is not available for `oedebugrt` to use since `oehost` is not a shared library to link against.

Hence, the code in `oedebugrt` avoids use of functions from rest of OE SDK.